.github/workflows/release-helper.yml

```
@@ -13,6 +13,25 @@ jobs:
         if: github.repository_owner != 'cmu-delphi'
         run: exit 1
 
+  client_changed:
+    needs: correct_repository
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: Get Python client file changes
+        id: changed-py
+        uses: dorny/paths-filter@v3
+        with:
+          filters: |
```

the unspecified base/ref usage seems it would be comparing `dev` to `main`, which is *probably* fine, but doing something based on whats in https://github.com/cmu-delphi/covidcast-indicators/blob/a6ea00355d79a47cc09db8090f6e5121c718de0f/.github/workflows/publish-release.yml#L31-L41 might be a little safer or at least more obvious?  idk. 
```suggestion
        with:
          # when base and ref are the same, changes are detected against the most recent commit before the push.
          # see https://github.com/dorny/paths-filter#usage for details.
          base: 'main'
          ref: 'main'
          filters: |
```